content/broker/broker-transition/usage/rest/_index.en.md

```diff
@@ -0,0 +1,207 @@
+{{% /notice %}}
+
+### Get File Details outbox (sender)
+
+This operation returns Broker File metadata.
+
+This request will not differ in usage between Altinn 2 and Altinn 3 transitioned services.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/outbox/{fileId} HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+### Get File Receipt outbox (sender)
+
+This operation returns Broker File Receipt
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/outbox/{fileId}/receipt HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+#### Example Receipt
+```JSON
+{
+    "ReceiptID": 0,
+    "ParentReceiptID": null,
+    "LastChanged": "2024-04-24T09:23:40.806",
+    "Status": "Ok",
+    "Text": "Upload of file 92a29085-5a63-4fc5-a294-bc2d4a331433 was successful. Recipients can now download the file.",
+    "SendersReference": null,
+    "ServiceOwnerPartyReference": null,
+    "PartyReference": "312903369",
+    "ReceiptHistory": null,
+    "SubReceipts": null
+}
+```
+#### Differences
+
+For Altinn 3 transitioned services the receipt is a pseudo receipt built from Altinn 3 Broker File metadata. The pseudo receipt will not contain subreceipts for the recipients.
+
+## Inbox (recipient)
+
+### Verify if users have files available
+This operation verifies whether any of the given recipients have any files available for download for the given ServiceCode / ServiceEdition combination.
+
+If files are available it returns true, otherwise it returns false.
+
+Implementation does not differ for Altinn 3 transitioned services.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/brokerservice/inbox/hasavailablefiles?serviceCode={serviceCode}&serviceEditionCode={serviceEdition}&recipients={recipient1},{recipient2} HTTP/1.1 
+accept: application/json
+ApiKey: myKey
+```
+
+### Get recipient inbox
+This operation retrieves a list of files available to the user.
+
+This operation will only return file list for transitioned services if the service is specified in the request.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/inbox/?serviceCode={serviceCode}&serviceEditionCode={serviceEdition} HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+#### Example receipt
+```JSON
+{
+    "ReceiptID": 0,
+    "ParentReceiptID": null,
+    "LastChanged": "2024-04-24T09:23:40.806",
+    "Status": "Ok",
+    "Text": "A file has been made available for download.",
+    "SendersReference": null,
+    "ServiceOwnerPartyReference": null,
+    "PartyReference": "313559017",
+    "ReceiptHistory": null,
+    "SubReceipts": null
+}
+```
+
+#### Differences
+The inbox operation will by default retrieve a list of Altinn 2 files. In order to retrieve Altinn 3 files, the service code and the service edition code parameters must be supplied.
```

Comment:
Consider rewording; if no service code/service edition code is supplied, the operation will only list files from Altinn 2.

Reply:
done